packages/design-system/src/components/TimePicker/__tests__/TimePicker.screen-object.tsx

```diff
@@ -0,0 +1,116 @@
+import { getByText, queryByText, render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { useState } from "react";
+import { TimePicker } from "..";
+import {
+  DropdownTimeInterface,
+  PeriodTypes,
+  TimePickerProps,
+} from "../TimePicker.interface";
+
+export type TimePickerTimeParts = "hours" | "minutes" | "seconds";
+export const mockErrorMessage = "Error";
+
+const TimePickerTestComponent = ({
+  ...props
+}: Omit<TimePickerProps, "time" | "setTime" | "defaultDropdownTime">) => {
+  const defaultDropdownTime = {
+    hours: 1,
+    minutes: "00",
+    seconds: "00",
+    period: PeriodTypes.AM,
+  };
+
+  const [time, setTime] = useState<DropdownTimeInterface>(defaultDropdownTime);
+
+  return (
+    <div className="max-w-[400px]">
+      <TimePicker
+        {...props}
+        time={time}
+        setTime={setTime}
+        defaultDropdownTime={defaultDropdownTime}
+      />
+    </div>
+  );
+};
+export class TimePickerObject {
+  get input(): HTMLInputElement {
+    return screen.getByPlaceholderText("Placeholder");
+  }
+  get supportingText() {
+    return screen.getByText("Supporting text");
+  }
+  get label() {
+    return screen.getByText("Label");
+  }
+  get error() {
+    return screen.getByText("Error");
+  }
+  get dropdown() {
+    return screen.getByTestId("timePickerDropdown");
+  }
+  get AMButton() {
+    return screen.queryByRole("button", { name: "AM" });
+  }
+  get PMButton() {
+    return screen.queryByRole("button", { name: "PM" });
+  }
+  get OKButton() {
+    return screen.getByRole("button", { name: "OK" });
+  }
+  get cancelButton() {
+    return screen.getByRole("button", { name: "Cancel" });
+  }
+
+  scroll(timePart: TimePickerTimeParts): HTMLUListElement {
```

Comment:
```suggestion
  getScroll(timePart: TimePickerTimeParts): HTMLUListElement {
```
This is a function, so we should use verbs as prefixes to imply that an action is taking place.